spec/models/tweet_spec.rb

```diff
@@ -1,4 +1,30 @@
 require 'rails_helper'
 
 RSpec.describe Tweet, type: :model do
+
+  describe '.default scope' do
+    it 'orders by most recently posted' do
+      tweet_one = FactoryGirl.create(:tweet)
+      expect(Tweet.first).to eq tweet_one
+    end
+  end  
+
+  describe 'validations' do
+    it "is valid with both user_id and text" do
+      expect(FactoryGirl.build(:tweet)).to be_valid    
+    end
+
+    it "must have text to be valid" do
+      expect(FactoryGirl.build(:tweet, user_id: 1, text: "")).to_not be_valid
```

Comment:
your tweet factory already has a user_id of 1, so i don't think passing it in here is necessary since it's the same.  passing in empty text, however, to test the validity (since you can't have blank tweets), is good  :)